### PR TITLE
[cyberkongz-v2] blacklist

### PR DESCRIPTION
**Pull request by frosti-eth**
Changes proposed in this pull request:
- removed liquidity pools from cyberkongz strategy

### CODE REVIEW

**Comment by frosti-eth** [comment]
One sec I need to retest

**Comment by frosti-eth** [comment]
ok good to go, thanks!

File: src/strategies/cyberkongz-v2/index.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'cesarsld';
```
Comment (ChaituVR) [inline src/strategies/cyberkongz-v2/index.ts:4]:
```suggestion
export const author = 'frosti-eth';
```

**Comment by cshlin** [comment]
Hello, anything else we need to do to get this merged?

**Comment by frosti-eth** [comment]
Hello! Anything else you need from me to get this merged? We're looking to use it asap :) Thanks!

**Review by bonustrack — APPROVED** [review]

**Pull request merged**